src/ServerPrivate.cc

```diff
@@ -318,7 +318,7 @@ void ServerPrivate::CreateEntities()
 void ServerPrivate::SetupTransport()
 {
   // Advertise available worlds.
-  std::string worldsService{"/gazebo/worlds"};
+  std::string worldsService{"/sim/worlds"};
```

Comment:
These services are user-facing, so we should at least tick-tock them. But in fact, I'm leaning towards keeping them as `gazebo`. Gazebo isn't technically wrong now, it's just less precise (i.e. refers to the entire stack instead of just the simulator). I think that's ok if it makes user's lives easier when migrating.

So let's leave all services and topics that use `gazebo` as they are.

Reply:
Reverted, same for `/sim/resource_paths`